app/src/main/res/values/styles.xml

```diff
@@ -656,10 +655,10 @@
     <style name="RecentBookmarks.FaviconCard" parent="Mozac.Widgets.Favicon">
         <item name="android:layout_width">@dimen/recent_bookmark_item_width</item>
         <item name="android:layout_height">@dimen/recent_bookmark_item_height</item>
-        <item name="android:padding">@dimen/recent_bookmark_item_padding</item>
+        <item name="android:padding">16dp</item>
         <item name="cardBackgroundColor">?mozac_widget_favicon_background_color</item>
-        <item name="cardCornerRadius">@dimen/recent_bookmark_item_card_radius</item>
-        <item name="cardElevation">@dimen/recent_bookmark_item_card_elevation</item>
+        <item name="cardCornerRadius">8dp</item>
+        <item name="cardElevation">@dimen/home_collection_elevation</item>
```

Comment:
```suggestion
        <item name="cardElevation">@dimen/home_card_elevation</item>
```

Card elevation since this isn't used for only a collection?

Reply:
The `home_collection_elevation` is what's being used for Jump back in, Collections, and this. We could change the name to make it more generic

Reply:
Yeah, sorry I should have been clearer: let's just change the name because it was a bit confusing why we were using something collections related here.